omniduct/databases/_cursor_formatters.py

```diff
@@ -9,14 +9,15 @@ class CursorFormatter(object):
 
     def __init__(self, cursor, **kwargs):
         self.cursor = cursor
+        self.lowercase_columns = kwargs.pop('lowercase_columns', False)
```

Comment:
What do you think about making this a more generic feature, such as "column_formatter", and allowing functions to be passed like `lambda x: x.lower()`? We could have a mapping of string defaults so that you can use the yaml configuration, including: {'lowercase': lambda x: x.lower()}.

Just thinking about future proofing this a bit and avoiding breaking people's code :).